rootfs/etc/cont-init.d/06-rtlsdr-usbreset

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/execlineb -P
```

Comment:
Have you tested this?

I don't normally write my run scripts using execlineb, so I don't know how this will behave...

When I write run scripts using bash, the s6-overlay will restart the run script if it exists. I'm worried that this script will be run over and over and over in a loop...

Reply:
It doesn't, since it's inside `cont-init.d` which is for oneshot scripts. I have tested it and it only runs once in my container.

Reply:
I just found this: https://github.com/just-containers/s6-overlay/issues/86#issuecomment-142199803

That comment indicates that it should behave as expected. Keen to know if your testing also shows this.

Reply:
> It doesn't, since it's inside `cont-init.d` which is for oneshot scripts. I have tested it and it only runs once in my container.

Perfect!